lib/challenge.js

```diff
@@ -136,16 +158,23 @@ function prompt7() {
 
 //  define someNumber here
 var someNumber;
```

Comment:
`var` is ok to use, but stick with `let` or `const` as that is more updated JavaScript syntax!